ote/src/clj/ote/services/pre_notices/pre_notices.sql

```diff
@@ -0,0 +1,7 @@
+-- name: fetch-authority-pre-notices
+SELECT p.id, p.sent, p.created, p.regions, p."route-description", p."pre-notice-type" as "pre-notice-type",
```

Comment:
Minor style remark: some SQL keywords here are lowercase instead of uppercase.

Reply:
Changed